src/reducers/userFormReducer.js

```diff
@@ -0,0 +1,60 @@
+export const GET_USER_FORM = "GET_USER_DATA";
```

Comment:
обычно пишут через префикс
```suggestion
export const USER_GET_FORM = "USER/GET_FORM";
```
это для саг? почему тут GET?

Reply:
`GET_USER_FORM` это когда сага успешно получит данные, диспатчит это екшн и данные залетают в стейт. Для самой саги экшн `TRIGGER_GET_USER_FORM`